examples/experimental/torch/classification/bootstrap_nas_search.py

```diff
@@ -0,0 +1,184 @@
+
+    set_seed(config)
+
+    # define loss function (criterion)
+    criterion = nn.CrossEntropyLoss()
+    criterion = criterion.to(config.device)
+
+    model_name = config['model']
+    train_criterion_fn = inception_criterion_fn if 'inception' in model_name else default_criterion_fn
+
+    nncf_config = config.nncf_config
+    pretrained = is_pretrained_model_requested(config)
+
+    # Data loading code
+    train_dataset, val_dataset = create_datasets(config)
+    train_loader, _, val_loader, _ = create_data_loaders(config, train_dataset, val_dataset)
+
+    bn_adapt_args = BNAdaptationInitArgs(data_loader=wrap_dataloader_for_init(train_loader), device=config.device)
+    nncf_config.register_extra_structs([bn_adapt_args])
+    # create model
+    model = load_model(model_name,
+                       pretrained=pretrained,
+                       num_classes=config.get('num_classes', 1000),
+                       model_params=config.get('model_params'),
+                       weights_path=config.get('weights'))
+
+    model.to(config.device)
+
+    validate(val_loader, model, criterion, config)
+
+    # define optimizer
+    params_to_optimize = get_parameter_groups(model, config)
+    optimizer, lr_scheduler = make_optimizer(params_to_optimize, config)
+
+    def train_epoch_fn(loader, model_, compression_ctrl, epoch, optimizer_):
+        train_epoch(loader, model_, criterion, train_criterion_fn, optimizer_, compression_ctrl, epoch, config,
+                    train_iters=config.train_steps, log_training_info=True)
+
+    def validate_model_fn(model_, loader):
+        top1, top5, loss = validate(loader, model_, criterion, config, log_validation_info=False)
+        return top1, top5, loss
+
+    def validate_model_fn_top1(model_, loader_):
+        top1, _, _ = validate_model_fn(model_, loader_)
+        return top1
+
+    nncf_network = create_nncf_network(model, nncf_config)
+
+    if config.search_mode_active:
+
+        compression_state = torch.load(config.search_elasticity_state_path)
+        model, elasticity_ctrl = resume_compression_from_state(nncf_network, compression_state)
+        model_weights = torch.load(config.search_supernet_weights)
+
+        load_state(model, model_weights, is_resume=True)
+
+        top1_acc = validate_model_fn_top1(model, val_loader)
+        print(f'SuperNetwork Top 1: {top1_acc}')
+
+        search_algo = SearchAlgorithm.from_config(model, elasticity_ctrl, nncf_config)
+
+        elasticity_ctrl, best_config, performance_metrics = search_algo.run(validate_model_fn_top1, val_loader, config.checkpoint_save_dir, tensorboard_writer=config.tb)
+
+        print(best_config)
+        print(performance_metrics)
+
+        search_algo.search_progression_to_csv()
+        search_algo.evaluators_to_csv()
+
+        top1_acc = validate_model_fn_top1(model, val_loader)
+        print(top1_acc, elasticity_ctrl.multi_elasticity_handler.count_flops_and_weights_for_active_subnet()[0]/2000000)
```

Comment:
please use nncf_logger with the appropriate indicator of this log

Reply:
Added logger. Thanks!